<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewport</class>
 <widget class="QMainWindow" name="Viewport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myRT Viewport</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="s_rednerSettingsGB">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>391</width>
      <height>379</height>
     </rect>
    </property>
    <property name="title">
     <string>Render Settings</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="14" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>BG Colour</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QSpinBox" name="m_filmHeight">
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="value">
        <number>720</number>
       </property>
      </widget>
     </item>
     <item row="11" column="3">
      <widget class="QDoubleSpinBox" name="m_ambientG">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QDoubleSpinBox" name="m_camTargetZ">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="m_camTargetY">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Camera Position</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="m_camUpX">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>AA Samples</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="m_camPosY">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="14" column="4">
      <widget class="QDoubleSpinBox" name="m_bgB">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QDoubleSpinBox" name="m_camUpZ">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Max Ray Bounces</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QDoubleSpinBox" name="m_ambientR">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QSpinBox" name="m_maxRayBounces">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Camera Up</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="m_fov">
       <property name="maximum">
        <double>175.000000000000000</double>
       </property>
       <property name="value">
        <double>80.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QDoubleSpinBox" name="m_camUpY">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="13" column="3">
      <widget class="QSpinBox" name="m_threadWidth">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="14" column="3">
      <widget class="QDoubleSpinBox" name="m_bgG">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="m_camTargetX">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Camera FOV</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QDoubleSpinBox" name="m_camPosZ">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ambient Light Col</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QCheckBox" name="m_multiThread">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QSpinBox" name="m_aa">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="11" column="4">
      <widget class="QDoubleSpinBox" name="m_ambientB">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="m_camPosX">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QDoubleSpinBox" name="m_bgR">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Multithreading</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Camera Target</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QSpinBox" name="m_filmWidth">
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="value">
        <number>1280</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Film Size</string>
       </property>
      </widget>
     </item>
     <item row="13" column="4">
      <widget class="QSpinBox" name="m_threadHeight">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>32</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>380</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_scenePath"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="m_render">
       <property name="text">
        <string>Render</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="m_loadScene">
       <property name="text">
        <string>load scene</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QProgressBar" name="m_renderProgress">
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>851</width>
      <height>651</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="m_graphicsLable">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>33</y>
       <width>821</width>
       <height>601</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
